toast-tk-runtime/pom.xml

```diff
@@ -48,7 +48,17 @@
 			<version>4.5.1</version>
 			<scope>test</scope>
 		</dependency>
-
+		<dependency>
+		    <groupId>javax.mail</groupId>

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)